src/components/breadcrumbs.cjsx

```diff
@@ -30,12 +30,16 @@ module.exports = React.createClass
       if i is @props.currentStep
         classes.push('current')
         classes.push('active')
+        bsStyle = 'primary'
         # classes.push('disabled')
         title = "Current Step (#{step.type})"
 
       if step.is_completed
         classes.push('completed')
-        bsStyle = 'primary'
+        if step.correct_answer_id is step.answer_id or not step.correct_answer_id
+          classes.push('btn-success')
```

Comment:
Should this just be `bsStyle = 'success'`? Is there a need for both `btn-primary` and `btn-success` to be on the element?